internal/client/response.go

```diff
@@ -37,7 +38,18 @@ func (r *ResponseError) Error() string {
 		return fmt.Sprintf("error parsing error response from server: %s", err)
 	}
 
-	return out.Message
+	if len(out.Message) > 0 {
+		return out.Message
+	}
+
+	switch r.statusCode {
+	case http.StatusBadRequest:
+		return "something went wrong"
```

Comment:
Why for a bad request we return only a `something went wrong` error? It should not be better to show also the response from the server?

Reply:
It's the case when the response is an error but it does not have the `message` field

Reply:
It could be something but not the message field? It's quite difficult for the user to retrieve what happen. Maybe we could think about a verbose mode. What do you think?

Reply:
We will keep it as an improvement consideration when we rework the log system to be more easy to use use and with more information